src/style.scss

```diff
@@ -57,16 +58,12 @@ $auro-inset-directions:'--squish';
   }
 }
 
-// handle focus-visible state
-:host(.focus-visible) {
-  .auro-button {
-    @include focus-visible;
+:focus-visible.auro-button {
```

Comment:
Chained selectors fails compliance standards outlined here: http://auro.alaskaair.com/support/css-conventions

Change to a single class such as `.focus-visible-auro-button` to improve render performance?

Reply:
Since :focus-visible is a pseudo-selector, it follows the [CSS conventions around state](https://auro.alaskaair.com/support/css-conventions#state).